### PR TITLE
Required checks

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: docs/Required-checks.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## Checks for the `master` branch
+
+Legend:
+* ✅ Required check.
+  Merges disallowed unless this job is successful.
+* 👉 Pre-requisite required check.
+  A required check depends on this job.  A job with a required check could not pass without this passing.
+* ⛔ No required check and not a pre-requisite.
+
+* Check Markdown links / markdown-link-check (push) ⛔
```
Comment (carbolymer) [inline docs/Required-checks.md:10]: emojis in on the left side would be more visible I think

**Review by carbolymer — APPROVED** [review]

**Comment by carbolymer** [comment]
It would be a good idea to link to this page from PR template

File: docs/Required-checks.md (hunk 1 above)
Comment (newhoggy) [reply docs/Required-checks.md]: Done!

**Comment by newhoggy** [comment]
Updated to reflect the new status-quo.

**Pull request merged**